### PR TITLE
Use a simple express server for local testing

### PR DESCRIPTION
**Pull request by mhuebert**
Uses a local express server for dev testing, much faster than the SAM lambda runtime.

also: 
- Home tab now works
- Refactored things into more namespaces

### CODE REVIEW

File: minimal-shadow-cljs-nodejs/src/server/main.cljs [hunk 1]
```diff
@@ -2,210 +2,88 @@
...
+  (p/try (p/let [response (cond
+                            ;; Slack API: identification challenge
+                            (:challenge body) (:challenge body)
+
+                            ;; Slack Interaction (e.g. global shortcut)
+                            (:payload body) (handlers/handle-interaction! (:payload body))
+
+                            ;; Slack Event
+                            (:event body)
+                            (handlers/handle-event! (:event body))
```
Comment (daveliepmann) [inline minimal-shadow-cljs-nodejs/src/server/main.cljs:41]: Just daydreaming on API design for handlers: `handler/event!`?

Comment (mhuebert) [reply minimal-shadow-cljs-nodejs/src/server/main.cljs:41]: Sounds good

**Comment by mhuebert** [comment]
> LGTM, only question is whether it runs in "prod" and what the current code in prod is based on.

Yeah, I think prod is still based on your AWS account? At least in the aws console I didn’t find anything under sparkboard, but could have looked in the wrong place in the multi-galaxy AWS reality.

**Comment by daveliepmann** [comment]
>I think prod lambdas are in your AWS account

I log in as `sparkboard-dave` and navigate to `Lambda > Functions > slackBot`; the URL I use is https://eu-central-1.console.aws.amazon.com/lambda/home?region=eu-central-1#/functions/slackBot?tab=configuration 

Is that visible to you?

**Comment by mhuebert** [comment]
Ah! it's visible, I didn't see it because I was in a different AWS region and did not realize the dashboard does not show all of the things.

**Pull request merged**